src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -189,6 +189,12 @@ export function PharmacyCard(props: PharmacyProps) {
         if (tag === "Email") {
           return <Badge status="info">{t("email")}</Badge>;
         }
+        if (tag === "1st Dose") {
```

Comment:
Generally when comparing a variable for many possible values, it is customary to use a `switch` statement.